assets/css/global.css

```diff
@@ -48,7 +48,7 @@
     */
     --text: var(--polar-night-dark);
     --text-muted: var(--polar-night-very-light);
-    --background: url('css/bg/light.svg');
+    --background: url('~css/bg/light.svg');
```

Comment:
Sorry but you **must** include the assets folder so the correct path would be `~assets/css/bg/light.svg`.

Reply:
ok